CMakeLists.txt

```diff
@@ -327,7 +327,9 @@ if(EMSCRIPTEN)
   endif()
 
   if("${CMAKE_BUILD_TYPE}" MATCHES "Release")
-    add_link_flag("-sSINGLE_FILE")
+    if(NOT BUILD_FOR_BROWSER)
+      add_link_flag("-sSINGLE_FILE")
+    endif()
```

Comment:
Maybe move this down to new line 362, after noderawfs? We might as well put all the node-related stuff together.

Reply:
Done in https://github.com/WebAssembly/binaryen/pull/6135/commits/ea692353bbba81361bb9085db4ee83d8c94e099f.